File: Ejercicios/01.SumaQtesuma/SumaQtesuma.c
#include <stdio.h>
#include <stdlib.h>

int main(){
    int suma, i, n, m;
    suma = 0;

    do{
        printf("Ingrese un numero mayor a 0\n");
        scanf("%i", &m);
        if (m <= 0){
            printf("Error, debe ser >0\n");
        }   
    } while (m <= 0);

    do{
        printf("Ingrese un numero mayor a 0\n");
        scanf("%i", &n);
        if (n <= 0){
            printf("Error, debe ser >0\n");
        }
    } while (n <= 0);  
     
    for (i = 0; i < n; i++){
        suma = suma + m;
    }

    printf("%i x %i = %i\n", m, n, suma);

    return EXIT_SUCCESS;
}

File: Ejercicios/01.SumaQtesuma/SumaQtesuma.py
def main():
    suma = 0
    
    while True:
        m = int(input("Ingrese un numero mayor a 0\n"))
        if m > 0:
                break
        else:
            print("Error, debe ser >0")
            
    while True:
        n = int(input("ingrese un numero mayor a 0\n"))
        if n > 0:
            break
        else:
            print("Error, debe ser >0 ")
            
    for i in range( 1, n + 1 ):
        suma = suma + m
    
    print(f"{m} x {n} = {suma}")
    
if __name__ == "__main__":
    main()